### DISCUSSION TITLE
Ignore time step in exodiff?

### DISCUSSION THREAD

**Discussion by aprilnovak** [idaholab/moose#16792] [Q&A General] [answered]
Hi all,

I'm trying to create a test that makes sure a pseudo-transient simulation reaches the same physical results regardless of whether a MOOSE-wrapped app is subcycled or has a time step that matches the master app's time step. To reach the same steady state tolerance, however, requires a different number of time steps (as expected) when different subcycling approaches are used. 

For my subcycled test, I'm reusing the gold file from the non-subcycled case. How can I ignore the time step for my Exodiff type test? I tried adding a blank `TIME STEPS` to a custom compare file like follows, but that didn't seem to do anything.

```
TIME STEPS

GLOBAL VARIABLES
  flux
  temperature
```

Basically, I want this test to pass b/c I don't care about the simulation time:

```
test:conduction/cylinders.subcycle: Sideset Distribution Factors:
test:conduction/cylinders.subcycle:   --------- Time step 1, 2.3000000e+00 ~ 3.5750000e+01, rel diff: -9.35664e-01 (FAILED)
test:conduction/cylinders.subcycle: Global variables:
test:conduction/cylinders.subcycle: Nodal variables:
test:conduction/cylinders.subcycle: Element variables:
test:conduction/cylinders.subcycle:
test:conduction/cylinders.subcycle: exodiff: Files are different
```

**Comment by loganharbour** [comment, marked as answer]
If it's just one timestep, you can use `-TM` in exodiff to compare the closest timesteps.

See page 13 for other ideas: https://ftp.mcs.anl.gov/pub/pdetools/nightlylogs/xsdk/xsdk-configuration-tester/packages/trilinos/packages/seacas/doc/exo_util.pdf

**Reply by aprilnovak** [reply]
Oh neat, I didn't know there was such good documentation on the settings for that! Unless I'm doing something really wrong, I don't think that works though, because my gold file has 23 time steps, but my test runs for 35 seconds, so I think that ends up comparing the results at time step 23 (which aren't converged yet for the other test).

I'll just copy the gold file, it's not a big deal - just was trying to be concise!
